File: library/src/androidTest/java/com/raizlabs/android/dbflow/test/structure/TestModel2.java
package com.raizlabs.android.dbflow.test.structure;

import com.raizlabs.android.dbflow.annotation.Column;
import com.raizlabs.android.dbflow.annotation.ModelContainer;
import com.raizlabs.android.dbflow.annotation.Table;
import com.raizlabs.android.dbflow.test.TestDatabase;

/**
* Author: andrewgrosner
* Contributors: { }
* Description:
*/
@Table(databaseName = TestDatabase.NAME)
@ModelContainer
public class TestModel2 extends TestModel1 {
    @Column(name = "model_order")
    int order;
}

File: dbflow-tests/src/test/java/com/raizlabs/android/dbflow/structure/TestModel2.kt
package com.raizlabs.android.dbflow.structure

import com.raizlabs.android.dbflow.annotation.Column
import com.raizlabs.android.dbflow.annotation.Table
import com.raizlabs.android.dbflow.TestDatabase

@Table(database = TestDatabase::class)
class TestModel2 : TestModel1() {

    @Column(name = "model_order")
    var order: Int = 0
}
